### DISCUSSION TITLE
No Huky temperature readout with my Phidget 1048_0 after restoring Artisan settings

### DISCUSSION THREAD

**Discussion by tomhusband** [artisan-roaster-scope/artisan#935] [Q&A] [answered]
I've been using my 1048_0 Phidget for several years on a Win 10 laptop. My roaster is a Huky. The other day the laptop died so I bought a new Win 10 unit, reinstalled Artisan and restored the Artisan .aset settings file. I installed the legacy Phidget drivers and the Phidget control panel shows my device. Artisan has all my settings but when I fire up the roaster I get only "half zeros" for BT, ET and MET and no graph at all. 

I've gone through the help files written by Susan on the old Huky Forum and things seem to be set up correctly. I'm kind of at a loss where to look further. It's probably something simple but any help would be appreciated.

**Comment by MAKOMO** [comment, marked as answer]
Why did you the legacy Phidget driver? That one is not supported any longer since v1.2.0 from 12/2017. Just install the current Phidget driver, use the Phidget Control Panel to update the firmware on your 1048 to its latest version and then run Artisan and do some roasts.

**Reply by tomhusband** [reply]
Yes, wrong assumption on my part. I uninstalled the old ones and installed the new ones. All is well now.

Thanks.
